### ISSUE TITLE
Help: Wrong magnetic field result for current dipole in spherical conductor

### ISSUE THREAD

**Issue by youyouhdhd** [openmeeg/openmeeg#608]
Hello everyone
First of all, thanks to the developer for providing such a handy toolkit, my problem with fieldtrip is as follows:
When I used the "openmeeg" method to calculate the meg-positive problem, I found some abnormal results, and the referenced fieldtrip link is as follows:
https://www.fieldtriptoolbox.org/workshop/oslo2019/forward_modeling/
I refer to the above tutorial to solve the magnetic field generated by the current dipole in the outer plane of the conductor in a spherical conductor, which is located at the center of the sphere, and verify the calculated leadfield, the results are shown in the figure below:
![Snipaste_2023-04-15_01-59-30](https://user-images.githubusercontent.com/54936458/232124812-7bb4649c-aebc-4b20-ae8f-cd1bfc590187.jpg)

![Snipaste_2023-04-15_01-57-54](https://user-images.githubusercontent.com/54936458/232124785-618ae82d-e0ee-4f10-a6ef-34eec666babc.jpg)

As you can see, the result is very unconventional, and I have used another calculation library to get a typical result of the magnetic field generated by the current dipole：
![Snipaste_2023-04-15_02-01-00](https://user-images.githubusercontent.com/54936458/232124876-cb20ca6b-b965-4d08-9a8e-19722db90f27.jpg)

so I do not know where the error occurred, I hope anyone can help me, I would be grateful.
PS: I calculated the EEG problem in the example using the "openmeeg" method, and the results are roughly similar to the correct results in the example in terms of the shape of the leadfield, but there are only differences of order of magnitude.
The data I used will be uploaded to google drive, thanks again!
https://drive.google.com/drive/folders/1RIuPS5kOC2OVcijLZ9j8BD0I3hcz4bix?usp=share_link

**Comment by larsoner** [comment]
Probably a duplicate of https://github.com/openmeeg/openmeeg/issues/577 ?

**Comment by youyouhdhd** [comment]
> Probably a duplicate of #577 ?

Thanks for your reply, I'm just a newbie, I read through the issue you mentioned and it seems to be a problem with openmeeg?

**Comment by larsoner** [comment]
Yes, I'm saying what you described in this issue might be the same bug with openmeeg described in #577

**Comment by youyouhdhd** [comment]
> Yes, I'm saying what you described in this issue might be the same bug with openmeeg described in #577是的，我是说你在这个问题中描述的可能与 #577 中描述的 openmeeg 的错误相同

I just tested using the triple nested ball model and the results are still weird, maybe I wrote it wrong? But I wrote it according to the fieldtrip example, what a headache
